.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,5 @@
+Describe the big picture of your changes here to communicate to the maintainers why we should accept this pull request.
+
+If it fixes a bug or resolves a feature request, be sure to link to that issue.
+
+The best way to propose a feature is to open an issue first and discuss your ideas there before implementing them.
```

Comment:
This template is kind of late after the fact. It's shown when a PR is being created. People have already done the work for the PR. I wouldn't want to tell them not to submit.

What it should be is a template for what the PR message should be. For that I don't have a good example.